crates/typst-library/src/math/mod.rs

```diff
@@ -82,8 +82,8 @@ use crate::text::TextElem;
 /// - Within them, Typst is still in "math mode". Thus, you can write math
 ///   directly into them, but need to use hash syntax to pass code expressions
 ///   (except for strings, which are available in the math syntax).
-/// - They support positional and named arguments, but don't support trailing
-///   content blocks and argument spreading.
+/// - They support positional and named arguments, as well as argument
+///   spreading, but don't support trailing content blocks.
```

Comment:
At this point, "they don't support trailing content blocks" is probably best as its own bullet.